create_secrets.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
```

Comment:
Most other files in this repo are hyphenated, can you call the `create-secrets.js`?